apps/bfd-server/bfd-server-docs/src/main/java/gov/cms/bfd/datadictionary/CsvToExcelCommand.java

```diff
@@ -0,0 +1,326 @@
+package gov.cms.bfd.datadictionary;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Set;
+import java.util.function.Consumer;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVRecord;
+import org.apache.poi.ss.usermodel.BorderStyle;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellStyle;
+import org.apache.poi.ss.usermodel.FillPatternType;
+import org.apache.poi.ss.usermodel.IndexedColors;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.VerticalAlignment;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.ss.util.CellRangeAddress;
+import org.apache.poi.xssf.usermodel.XSSFColor;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+/**
+ * Functions to create and format an Excel xlsx workbook from an existing set of CSV files for the
+ * V1 and V2 versions of the BFD data dictionary.
+ */
+public class CsvToExcelCommand {
+
+  /** Excel columns widths in units of 1/256th of a character width for the V1 data dictionary. */
+  private static int[] v1ColumnsWidths =
```

Comment:
final ?

Reply:
Agreed and a `List` so that the values are immutable.

```java
private static final List<Integer> v1ColumnWidths = List.of(...);
```

Reply:
Yep, should be final and immutable